### PR TITLE
Add datetime string format

### PR DESCRIPTION
**Pull request by igarcez**
This MR formats the datetime string like `"2020-03-03T00:00:00.000000+0000"`

### CODE REVIEW

**Comment by tonythomas01** [comment]
Do we really need the milliseconds ? Can we follow `2020-04-04T19:29:31+00:00` ? https://en.wikipedia.org/wiki/ISO_8601 

and we might want to update the API docs in docs/ and sadly both the yaml and the json. Well, you can edit the yaml and autogenerate the json from https://editor.swagger.io/

**Comment by igarcez** [comment]
there are a few pointers here,
I couldn't figure out how to make the timezone to look like "+00:00" instead it looks like "+0000"
on my test, the endpoint result page changes a bit

Screen Shot 2020-04-04 at 9 15 50 PM

to 

Screen Shot 2020-04-04 at 9 16 03 PM

**Comment by igarcez** [comment]
@tonythomas01 should be okay to remove the milliseconds

File: quarantined_backend/local_settings_sample.py [hunk 1]
```diff
@@ -23,6 +23,11 @@
 STATIC_URL = "/static/"
```
Comment (tonythomas01) [inline quarantined_backend/local_settings_sample.py:23]: this should go to settings.py, btw.

**Review by tonythomas01 — COMMENTED** [review]
We should have this in settings.py. Lets see if we can get the ISO format. Else, we go with this.

**Comment by tonythomas01** [comment]
> @tonythomas01 should be okay to remove the milliseconds

Apparently this is the best we got :) Lets go for it then!

**Comment by igarcez** [comment]
closing after the slack sync up, conclusion is that this is not desireable anymore

**Pull request closed without merging**